src/main/java/peoplesoft/logic/commands/ExportCommand.java

```diff
@@ -23,11 +23,17 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": Exports a .csv file with the jobs the person worked on, "
             + "including how much pay they should expect to receive.\n"
-            + "Format: " + COMMAND_WORD + " INDEX. Example: " + COMMAND_WORD + " 1";
+            + "All the jobs the person worked on will be displayed under the Jobs list.\n"
+            + "Format: " + COMMAND_WORD + " INDEX. \n"
+            + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_EXPORT_PERSON_SUCCESS = "Exported Person Details: %1$s";
+    public static final String MESSAGE_EXPORT_PERSON_SUCCESS = "%s's details were "
+            + "exported to a .CSV in your data folder using their name. \n"
+            + "Now displaying the jobs that they were assigned to.\n"
+            + "use the \"list\" command to see all jobs again.";
```

Comment:
it's a new sentence, right?

```suggestion
            + "Use the \"list\" command to see all jobs again.";
```

Reply:
good catch